.gitignore

```diff
@@ -3,3 +3,6 @@ coverage
 node_modules
 .DS_Store
 coverage/
+*.js
+!test/*.js
```

Comment:
After this change, is there still a risk to get unwanted `.js` files?

Reply:
And without those two lines in the `.gitignore`, is there a risk to get unwanted JS files?

Reply:
Well yes, if you `tsc` locally

Reply:
Ok, so LGTM! 👍